draft-ietf-tls-exported-authenticator.md

```diff
@@ -222,9 +228,10 @@ excludes all RSASSA-PKCS1-v1_5 algorithms and combinations of ECDSA and hash
 algorithms that are not supported in TLS 1.3.
 
 If an authenticator request is present, the signature algorithm MUST be chosen
-from one of the signature schemes in the authenticator request. Otherwise, the
-signature algorithm used should be chosen from the "signature_algorithms"
-extension sent by the peer in the TLS handshake.
+from one of the signature schemes in the authenticator request.  Otherwise, the
+signature algorithm used should be chosen from the "signature_algorithms_cert"
+extension (in TLS 1.3) or "signature_algorithms" (in TLS 1.2) sent by the peer
+in the TLS handshake.
```

Comment:
I think that the signature_algorithms extension is what is relevant here.  the _cert variant only really affects the keys on the chain up to, but not including the EE.